src/regions/ca/info/ca-bc/clinics.md

```diff
@@ -1,11 +1,5 @@
-## British Columbia: assessment
+## British Columbia: self-assessment tool
 
 [This self-assessment tool](https://covid19.thrive.health/), developed by the BC Ministry of Health, will help determine whether you may need further assessment or testing for COVID-19. You can complete this assessment for yourself, or on behalf of someone else, if they are unable to.
```

Comment:
I'd remove the indentation here

Reply:
weird it's only showing on the code preview